src/index.js

```diff
@@ -40,11 +40,11 @@ async function run() {
   }
 
   // Find the two most recent tags
-  const { data: tags } = await octokit.rest.repos.listTags({
-    owner,
-    repo,
-    per_page: 10,
-  });
+  // Using https://octokit.github.io/rest.js/v19#pagination
+  const { data: tags } = await octokit.paginate("GET /repos/{owner}/{repo}/tags",
+    { owner, repo },
+    response => response.data.map(d => d.name)
```

Comment:
The map needs to be removed, as the full tag object is needed.

Reply:
Yes I jsut realized that sorry, I fixed it

Reply:
Nice find with the paginate method though!